sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_client.py

```diff
@@ -82,11 +88,18 @@ def create_key(self, name, key_type, **kwargs):
         enabled = kwargs.pop("enabled", None)
         not_before = kwargs.pop("not_before", None)
         expires_on = kwargs.pop("expires_on", None)
-        if enabled is not None or not_before is not None or expires_on is not None:
-            attributes = self._models.KeyAttributes(enabled=enabled, not_before=not_before, expires=expires_on)
+        exportable = kwargs.pop("exportable", None)
+
+        if enabled is not None or not_before is not None or expires_on is not None or exportable is not None:
+            attributes = self._models.KeyAttributes(
+                enabled=enabled, not_before=not_before, expires=expires_on, exportable=exportable
```

Comment:
Why is the lvalue for `expires` not the same as `expires_on`. All the other properties match. Is this correct?

Reply:
The generated KeyAttributes model accepts an `expires` kwarg, but we rename this as `expires_on` as a method kwarg and as a property of KeyProperties. This is correct; it just looks a little weird